dali/test/python/test_dali_tf_dataset.py

```diff
@@ -77,11 +78,9 @@ def define_graph(self):
         if self.device is 'gpu':
             im_ids = im_ids.gpu()
         im_ids_16 = self.cast(im_ids)
+        reshaped = self.reshape(im_ids) + 0
```

Comment:
why `+ 0`?

Reply:
It's leftover from WAR for reshape not working. I can remove it now.